Update README and plugin.json docs links

imatwawana opened the pull request:
Updates README and plugin.json file with addition of docs on PR https://github.com/grafana/website/pull/28493
Don't merge until website PR is merged

imatwawana committed:
Added docs and sources folders, make files, markdown files, fixed front matter and other general fixes

imatwawana committed:
Fixed front matter

imatwawana committed:
Made edits based on review against Google style guide via copilot

imatwawana committed:
Updated links

imatwawana committed:
Updated README and plugin.json links

imatwawana committed:
Added deprecation admonition to index and README

imatwawana committed:
Added links to Business Text docs

imatwawana commented on README.md:
```diff
@@ -8,13 +8,12 @@
 [![CodeQL](https://github.com/VolkovLabs/business-news/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/VolkovLabs/business-news/actions/workflows/codeql-analysis.yml)
 
-## Announcement
+{{< admonition type="caution" >}}
```
@jdbaldry Just checking, given all the places this README is used, is it safe to use an admonition here or should I just use plain text and a heading?

jdbaldry replied:
I would be tempted to stick with plain text here

imatwawana commented on README.md:
```diff
@@ -8,13 +8,12 @@
 [![CodeQL](https://github.com/VolkovLabs/business-news/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/VolkovLabs/business-news/actions/workflows/codeql-analysis.yml)
 
-## Announcement
+{{< admonition type="caution" >}}
```
```suggestion
## Deprecation notice

```

imatwawana commented on README.md:
```diff
@@ -8,13 +8,12 @@
-### Why This Change?
-
-The retirement aligns with industry trends toward more integrated and scalable data solutions, addressing the complexities of managing fragmented data sources. The Infinity data source offers enhanced flexibility, supporting a wide range of data formats and real-time updates, making it an ideal choice for business analytics and dashboards.
+{{< /admonition >}}
```
```suggestion
```

imatwawana committed:
Replaced admonition with plain text

imatwawana committed:
Removed markdown files

imatwawana commented:
@jdbaldry I removed the Markdown files. Should I be nuking the entire `docs` folder?

Also, I've added Robby as a reviewer on the website PR, but he's not in this repo, so I couldn't add him on this one.

imatwawana committed:
Removed docs folder and makefiles

imatwawana commented:
@adela-almasan I've removed the docs changes from this PR since they're being added via a `website` repo PR. But I don't know why the zizmor check is failing. Can you take a look?
